db/fixtures/server_roles.csv

```diff
@@ -9,6 +9,7 @@ ems_metrics_coordinator,Capacity & Utilization Coordinator,1,false,zone
 ems_metrics_processor,Capacity & Utilization Data Processor,0,false,zone
 ems_operations,Provider Operations,0,false,zone
 event,Event Monitor,1,false,zone
+git_owner,GIT Repositories Owner,1,false,zone
```

Comment:
Git should not be in all caps.  I didn't mention it in the other PRs, but as this part is UI facing, it needs to be correct.